### PR TITLE
Check whether a number is a Strong Number or not in C++

### PR DESCRIPTION
**Pull request by Aayushi-Mittal**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes, I have

### Description

A program to check whether a number is strong number or not in C++.
A number can be called strong number if the sum of factorial of individual digits is equal to the number.
For Example - 145 is a strong number as 1!+4!+5!=145

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#4840

### CODE REVIEW

File: C-Plus-Plus/math/Strong_Number.cpp [hunk 1]
```diff
@@ -0,0 +1,42 @@
+//Strong number is a special number whose sum of factorial of digits is equal to the original number.
+//For example: 145 is strong number. Since, 1! + 4! + 5! = 145
+
+#include <iostream>
+using namespace std;
+int main()
+{
+    long long int num, n, factorial=1, sum=0;
+    int i, digit;
+    cin>>num;
```
Comment (siddharth25pandey) [inline C-Plus-Plus/math/Strong_Number.cpp:11]: Display message in print form as what are you taking input from the user

File: C-Plus-Plus/math/Strong_Number.cpp [hunk 2]
```diff
@@ -0,0 +1,42 @@
+//Strong number is a special number whose sum of factorial of digits is equal to the original number.
+//For example: 145 is strong number. Since, 1! + 4! + 5! = 145
+
+#include <iostream>
```
Comment (siddharth25pandey) [inline C-Plus-Plus/math/Strong_Number.cpp]: Use **bits/stdc++**

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Aayushi-Mittal Do the required changes

**Comment by Aayushi-Mittal** [comment]
@siddharth25pandey Made the required changes.

File: C-Plus-Plus/math/Strong_Number.cpp [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+/*
+Time Complexity: O(n*9)
+Space Complexity: O(1)
+
+sample 1:
+Input: 145
+Output: 145 is a strong number.
+sample 2:
+Input: 2
+output: 2 is not a strong number.
```
Comment (siddharth25pandey) [inline C-Plus-Plus/math/Strong_Number.cpp]: Sample I/O is **not matching** with the execution of the Program

* Order of Printing message is missing in Sample I/O

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/math/Strong_Number.cpp (hunk 3 above)
Comment (Aayushi-Mittal) [reply C-Plus-Plus/math/Strong_Number.cpp]: @siddharth25pandey Made the required changes. Please check now.

**Review by siddharth25pandey — APPROVED** [review]
@Aayushi-Mittal LGTM 💯 Able to merge

**Comment by siddharth25pandey** [comment]
@all-contributors please add @Aayushi-Mittal for her code

**Pull request merged**